test/load/allocation/README.md

```diff
@@ -1,20 +1,30 @@
-# Load Test for gRPC allocation service
+# Load Tests for gRPC Allocation Service
 
-This load tests aims to validate performance of the gRPC allocation service.
+[Allocation Load Test](#allocation-load-test) and [Scenario Tests](#scenario-tests)
+for testing the performance of the gRPC allocation service.
+
+## Prerequisites
 
-## Kubernetes Cluster Setup
+1. A [Kubernetes cluster](https://agones.dev/site/docs/installation/creating-cluster/) with [Agones](https://agones.dev/site/docs/installation/install-agones/)
+    - We recommend installing Agones using the [Helm](https://agones.dev/site/docs/installation/install-agones/helm/) package manager.
+    - If you are running in GCP, use a regional cluster instead of a zonal cluster to ensure high availability of the cluster control plane.
+    - Use a dedicated node pool for the Agones controllers with multiple CPUs per node, e.g. `e2-standard-4'.
+    - For Allocation Load Test:
+      - In the default node pool (where the Game Server pods are created), 75 nodes are required to make sure there are enough nodes available for all game servers to move into the ready state. When using a regional cluster, with three zones with the region, that will require a configuration of 25 nodes per zone.
+    - For Scenario Tests:
+      - [Kubernetes Cluster Setup for Scenario Tests](#kubernetes-cluster-setup-for-scenario-tests)
```

Comment:
nit: See [Kubernetes Cluster Setup for Scenario Tests](#kubernetes-cluster-setup-for-scenario-tests)